src/libs/fileDownload/FileUtils.ts

```diff
@@ -285,6 +286,27 @@ function isHighResolutionImage(resolution: {width: number; height: number} | nul
     return resolution !== null && (resolution.width > CONST.IMAGE_HIGH_RESOLUTION_THRESHOLD || resolution.height > CONST.IMAGE_HIGH_RESOLUTION_THRESHOLD);
 }
 
+const getImageDimensionsAfterResize = (file: FileObject) =>
+    ImageSize.getSize(file.uri ?? '').then(({width, height}) => {
+        let newWidth;
+        let newHeight;
+        if (width < height) {
+            newHeight = CONST.MAX_IMAGE_DIMENSION;
+            newWidth = (newHeight * width) / height;
+        } else {
+            newWidth = CONST.MAX_IMAGE_DIMENSION;
+            newHeight = (newWidth * height) / width;
+        }
```

Comment:
We can simplify this a bit

```suggestion
        const scaleFactor = CONST.MAX_IMAGE_DIMENSION / (width < height ? height : width);
        const newWidth = width * scaleFactor;
        const newHeight = height * scaleFactor;
```

Reply:
Thanks your suggestion